### PR TITLE
Portfolio accessibility

### PR DESCRIPTION
**Pull request by clin2on3mun**
**In this PR we add the following to the branch:**
> ### **Page titles:** 
The page title had been modified from personal Portfolio to Munana Clinton | Portfolio for a proper description.
> ### Image text alternatives : 
     > Add alt attributes to image tag for proper description >

 > ### Text headings :
      >The h1 tag is used on the heading in this section.

 > ### Color contrast 
   > 18 issues were found at first but we managed to fix all the color contrast issues. And manage to change the color contrast of all the sections.  

>### Resize
   >Resize was made on the texts.

 > ### Interaction 
      > No changes are required in this section. 

 > ### Moving content 
      >No changes are required in this section. 

> ### Multimedia 
      > No changes are required in this section. 

 > ### The basic structure of the page 
     > No issues were found here.

### CODE REVIEW

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi @clin2on3mun , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
-  No linter errors :clap: 
- Descriptive PR summary :+1: 
- Good use of accessibiliy rules :+1: 

## Required Changes ♻️
- [ ] Kindly make sure there is no contract to the button's interactions:
Screenshot from 2023-01-31 19-42-58


- [ ] I think it would be better to make sure the text is not overlapping while we are still between 375-1440px of width, as you can see below, at some point the skills list overlap :

Screenshot from 2023-01-31 20-20-23
Screenshot from 2023-01-31 20-19-57


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 1]
```diff
@@ -676,6 +681,10 @@
     border: 1px solid #ff6b00;
   }
 
+  .color:hover {
+    cursor: pointer;
+  }
+
   .title-line {
     width: 794px;
     height: 0;
```
Comment (brainconnect93) [inline styles.css]: - [ ] As suggested by the previous reviewer, I would kindly suggest you work on the contract to the button's interactions:
Currently your `see project` button is failing the button contrast checker test.

The result 👇 

but

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
# STATUS: Changes Required :recycle:

### Hi @Team, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (clin2on3mun) [reply styles.css]: thank you for reviewing my PR

**Review by victorokoroji — APPROVED** [review]
Hi @clin2on3mun, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**